modules/test/playwright/tests/change-tracking-web/unsupportedPublicationsScope.spec.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {changeTrackingPagesTest} from '../../fixtures/changeTrackingPagesTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+
+export const test = mergeTests(
+	featureFlagsTest({
+		'LPS-164563': true,
+	}),
```

Comment:
This is not the right feature flag. The original feature flag was removed in https://github.com/liferay/liferay-portal/commit/fb9b2167610f030692a3e91521f0356f425d62be, so we don't need to set it.